Bottom: 0639a2332527fbc85555a1f2047c362495e8ec1e
Top:    7413d85582079678a2245ca49e793f0cd8ad5bd2
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-31 10:18:27 -0700

Refresh of addrinfo-af-ib

---

diff --git a/man/ucmatose.1 b/man/ucmatose.1
index baf5837..73477ea 100644
--- a/man/ucmatose.1
+++ b/man/ucmatose.1
@@ -4,10 +4,10 @@ ucmatose \- RDMA CM connection and simple ping-pong test.
 .SH SYNOPSIS
 .sp
 .nf
-\fIucmatose\fR [-s server_address] [-b bind_address] [-f address_format]
-		[-c connections] [-C message_count] [-S message_size]
-\fIucmatose\fR -s server_address [-b bind_address] [-f address_format]
-		[-c connections] [-C message_count] [-S message_size] [-t tos]
+\fIucmatose\fR [-s server_address] [-b bind_address] [-c connections]
+		[-C message_count] [-S message_size]
+\fIucmatose\fR -s server_address [-b bind_address] [-c connections]
+		[-C message_count] [-S message_size] [-t tos]
 .fi
 .SH "DESCRIPTION"
 Establishes a set of reliable RDMA connections between two nodes using the
@@ -22,13 +22,6 @@ This option must be specified by the client.
 \-b bind_address
 The local network address to bind to.
 .TP
-\-f address_format
-Specifies the format of the server and bind address.  Be default, the
-format is determined by getaddrinfo() as either being a hostname, an IPv4
-address, or an IPv6 address.  This option may be used to indicate that
-a specific address format has been provided.  Supported address_format
-values are: name, ip, ipv6, and gid.
-.TP
 \-c connections
 The number of connections to establish between the client and server.
 (default 1)
